<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1520</width>
    <height>800</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_gimbaldisplay">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>90</y>
      <width>261</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>云台角度和转速显示区域</string>
    </property>
    <widget class="QLabel" name="label_horizontal">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>40</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>水平角度：</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_horizontal">
     <property name="geometry">
      <rect>
       <x>149</x>
       <y>30</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_vertical">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>俯仰角度：</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber_vertical">
     <property name="geometry">
      <rect>
       <x>149</x>
       <y>70</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Filled</enum>
     </property>
    </widget>
    <widget class="QSlider" name="slider_speed_vertical">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>241</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>8</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="slider_speed_horizontal">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>241</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>8</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_speed_vertical_control">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>垂直速度控制：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_speed_horrizontal_control">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>水平速度控制:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_gimbalcontrol">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>550</y>
      <width>261</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>云台方向控制区域</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_playerVolSet">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>311</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>多媒体音量设置区域</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_PlayVol">
       <property name="text">
        <string>播放音量：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSlider" name="Slider_PlayVolume">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_CalVol">
       <property name="text">
        <string>采集音量：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="Slider_CapVolume">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>311</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>指示灯区域</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_bird_repellent_switch_led">
       <property name="text">
        <string>led_bird_repellent</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_broadcast_led">
       <property name="text">
        <string>led_broadcast</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>驱离</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>喊话</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>311</width>
      <height>511</height>
     </rect>
    </property>
    <property name="title">
     <string>音频文件显示区域</string>
    </property>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>301</width>
       <height>301</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_listUpdate">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>420</y>
       <width>90</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string>列表更新</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnRepeatPlay">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>420</y>
       <width>90</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string>单曲循环</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPlay">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>420</y>
       <width>90</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string>播放</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPause">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>330</y>
       <width>90</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string>暂停/继续</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPrevious">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>330</y>
       <width>90</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string>上一曲</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnNext">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>330</y>
       <width>90</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string>下一曲</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>300</y>
      <width>261</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>云台和车头相对位置显示区域</string>
    </property>
    <widget class="QLabel" name="label_horizontal_pixmap">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>130</y>
       <width>53</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>photo</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>500</y>
      <width>451</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>强光频率、等级</string>
    </property>
    <widget class="QLabel" name="label_Spotlight_led">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>53</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Dazzling_led">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>53</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>53</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>探照</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>53</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>眩目</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>强光频率：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Frequency">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>53</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_switch_strongLight">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>强光开关</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_switch_flash">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>160</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>闪烁开关</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_angle_min">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>最小发散角</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_angle_max">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>200</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>最大发散角</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>261</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>驱离音与喊话区域</string>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>一键喊话</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnBroadcast">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>button</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>500</y>
      <width>91</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>模块设置区域</string>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>强声设置</string>
     </property>
    </widget>
    <widget class="QPushButton" name="GimbaOptionPage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>云台设置</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>强光设置</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbMediaPlayer">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>0</y>
      <width>550</width>
      <height>490</height>
     </rect>
    </property>
    <property name="title">
     <string>视频显示区域</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbOwWarnning">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>9</y>
      <width>350</width>
      <height>741</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1520</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
